smtp/smtp.js

```diff
@@ -129,25 +129,26 @@ SMTP.prototype = {
       }
     };
 
-    var response = function(err, msg) {
+    var cbs = 0, response = function(err, msg) {
```

Comment:
if you don't mind some stylistic nits, i'll do the other half of the work and make a new npm package and update this.

1) can you separate cbs and response on different lines?

```
var callbacks = 0
var respones = function ...
```

also, can you name cbs to callbacks?